trunk/SUAVE/Components/Wings/Segment.py

```diff
@@ -23,7 +23,9 @@ def __defaults__(self):
         self.dihedral_outboard     = 0.0
         self.sweeps                = Data()
         self.sweeps.quarter_chord  = 0.0
+        self.sweeps.leading_edge   = 0.0 # Added by Matthew
```

Comment:
Yeah I only did this to highlight changes if the code breaks (This was early before I realized the power of git)